### DISCUSSION TITLE
Is there a direct command to get the last m_n.jpeg file from the last folder?

### DISCUSSION THREAD

**Discussion by Rochan2001** [libvips/libvips#2263] [Q&A] [answered]
My last folder contains over 100,000 images. I wanted to know if there's a command which can tell me the rows and column of the last folder.I need the m,n from the last jpeg. Sorry if I'm not able put this in a more clear way.

**Comment by jcupitt** [comment, marked as answer]
Sorry, no. You'll need to use something like bash / psh / python to scan the files.

**Reply by jcupitt** [reply]
In bash, for example:

```
$ mkdir x
$ cd x
$ touch touch 12_{1..100000}.jpg
$ ls *.jpg | sort -rn -t _ -k2 | head -n 1
12_100000.jpg
$ 
```

(list files, sort in reverse numeric order, use `_` as the field separator, use the second field as the key, take the first item)

**Reply by Rochan2001** [reply]
thanks for the quick response, consider this solved :)

**Reply by Rochan2001** [reply]
i think i have to do this in python because it's showing argument list too long
